sample/set_cluster_location.py

```diff
@@ -0,0 +1,5 @@
+import rubrik_cdm
+
+RubrikConn = rubrik_cdm.Connect()
+
+RubrikConn.set_cluster_location("Example Location")
```

Comment:
`RubrikConn.set_cluster_location("Example Location")` to `rubrik.set_cluster_location("Example Location")`